src/App.css

```diff
@@ -1,33 +1,95 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 .App {
-  text-align: center;
+  max-width: 100%;
+  display: grid;
+  grid-template-rows: 50px 1fr;
+  grid-template-columns: 1fr;
+  width: 100vw;
+  height: 100vh;
+}
+
+.header {
+  display: flex;
+  background-color: #bfd3c1;
+  justify-content: center;
+  align-items: center;
+  position: sticky;
+  top: 0;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+.main-grid {
+  display: grid;
+  grid-template-rows: 80px 1fr;
+  grid-template-columns: 1fr;
+}
+.filter-section {
+  display: inline-flex;
+  flex-direction: row;
+  background-color: #68a691;
+  justify-content: space-between;
+  align-items: center;
+  overflow-x: auto;
+  white-space: nowrap;
+  width: 100vw;
+  position: sticky;
+  top: 50px;
```

Comment:
This value is not the same as the 80px of the first grid row. So it will probably overlap, when user scrolls up. Is this intentional?

Reply:
It doesn't overlap since 80px refers to the filter box height, the sticky position starts at 50px (just below header) and is then 80px in height :) ty!